dataprofiler/data_readers/avro_data.py

```diff
@@ -65,7 +65,16 @@ def is_match(cls, file_path, options=None):
         if options is None:
             options = dict()
 
+        # get current position of stream
```

Comment:
currently. this is the only `is_match` which takes into account seek. Do the other `is_match` functions not need it?